src/main/java/core/mvc/tobe/AnnotationHandlerMapping.java

```diff
@@ -16,12 +29,52 @@ public AnnotationHandlerMapping(Object... basePackage) {
     }
 
     public void initialize() {
+        Reflections reflections = new Reflections(basePackage);
+        reflections.getTypesAnnotatedWith(Controller.class)
+                .forEach(this::findRequestMappingMethods);
+
+    }
+
+    private void findRequestMappingMethods(Class<?> aClass) {
+        try {
+            Method[] methods = aClass.getDeclaredMethods();
+            Object instance = aClass.getDeclaredConstructor().newInstance();
+            Stream.of(methods).forEach(method -> addHandlerExecutions(instance, method));
```

Comment:
addHandlerExecutions안의 조건절을 Stream API 내부로 이동시켜도 되지않을까요
```suggestion
            Stream.of(methods)
                    .filter(method -> method.isAnnotationPresent(requestMappingClass))
                    .forEach(method -> addHandlerExecutions(instance, method));
```

Reply:
Stream.foreach 를 for loop으로 수정하면서 기존 위치에 두었습니다.